code/modules/cargo/exports/manifest.dm

```diff
@@ -1,7 +1,7 @@
 // Approved manifest.
 // +80 credits flat.
 /datum/export/manifest_correct
-	cost = CARGO_CRATE_VALUE * 0.4
+	cost = CARGO_CRATE_VALUE * 0.25
```

Comment:
why are you touching this. this doesnt seem related to the other changes

Reply:
I talked with other coders in the discord who are involved in economy code (I believe it was @ArcaneDefence) and they recommended I reduce the cost of the manifest to balance the extra credits you get from the requisition form.  If you return both paperwork properly you still get more than before the PR but only slightly. (both forms now give 100 credits vs manifest form give 80 credits before this PR)

Reply:
you should change the note above that says "+80 credits flat" to reflect this change

Reply:
i would have preferred that you split the cargo changes into another pr since youre bundling it with several other at best sorta related changes so it makes it hard to discuss its own merits.

Reply:
This is a paperwork and bureaucracy PR bundled together.  Stamped requisition forms falls into that category easily.  I could split this PR into 10 smaller ones for each additional thing but then it kind of ruins the fun and purpose of it being a DLC.  

---

### Before my changes:
- Manifest +80 credits
- Req form +0 credits

### After my changes:
- Manifest +50 credits
- Req form +50 credits

---

Initially I was just going to have the req forms give the same amount of credits as the manifest but I was convinced to lower both of them so it doesn't unbalance crate returns too much.  If you feel that this is too big of a balance change feel free to slap on the balance label.

Reply:
Cargo changes have been split into PR #66230